tests/checker/good/gold/adt_test.scilla.gold

```diff
@@ -75,10 +75,20 @@
   },
   "warnings": [
     {
-      "warning_message": "Unused library ADT: TestType1",
```

Comment:
Here is an interesting case. We see that ADT is used, but none of its constructors are used. So, the tool reports each constructor as unused.

This behavior seems logical to me. Probably, we could suppress warnings in such cases at all, because type is *somehow* used. But I don't see a way to reproduce this case in working code that could be compiled.

Reply:
That's a good point, I was surprised at first, but then thought if the user decided to remove `TestType1`, they would still get the right message.